docs/add_release_vintage.md

```diff
@@ -0,0 +1,203 @@
+# Add a vintage Workload Cluster template (Legacy CRs)
```

Comment:
```suggestion
# Add a vintage Workload Cluster template
```
I think we should drop the legacy lingo. Customers don't know it and internally we use vintage.